.github/workflows/opentelemetry.yml

```
@@ -0,0 +1,19 @@
+---
+name: OpenTelemetry Export Trace
+
+on:
+  workflow_run:
+    workflows:
+      - ci
+      - e2e
+    types: [completed]
+
+jobs:
+  otel-export-trace:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/apm-pipeline-library/.github/actions/opentelemetry@main
```

We use `main` only in the apm-pipeline-library so we can verify things are in a good shape before running the releases

```suggestion
      - uses: elastic/apm-pipeline-library/.github/actions/opentelemetry@current
```

---

Aw. Thanks. Copy pasta error.